### PR TITLE
docs(competition): Competition clarification

### PR DESCRIPTION
**Pull request by Matvey-Kuk**
Clarified docs about competition and positioning.

### CODE REVIEW

File: docs/overview/comparison.mdx [hunk 1]
```diff
@@ -4,38 +4,36 @@ title: "Comparison"
 
 Sometimes, it's easier to understand a tool's capabilities by comparing it to other tools in the same ecosystem. In this section, we'll discuss how Keep interacts with and compares to other tools in the ecosystem.
 
-## Keep vs Alerta
+## Keep vs AIOps inside observability platforms (Splunk ITSI, Elastic AIOps, etc.)
```
Comment (talboren) [inline docs/overview/comparison.mdx:7]: How about:
```suggestion
## Keep vs observability traditional AIOps
```
It feels a little to much for a title?

File: docs/overview/comparison.mdx [hunk 2]
```diff
@@ -4,38 +4,36 @@ title: "Comparison"
...
+| ------------------------------------- | -------------------------------------------------------------- | ---------------------------- |
+| Aggregation of alerts from one platform | ✅                                                           | ✅                            |
+| .. from multiple platforms            | ✅                                                             | ❌                            |
+| Correlation of alerts between multiple sources | ✅                                                    | ❌                            |
+| Alerts enrichment                     | ✅                                                             | ❌                            |
+| Open source                           | ✅                                                             | ❌                            |
+| Workflow automation                   | ✅                                                             | ❌                            |
 
-## Keep vs Grafana
+## Keep vs dedicated AIOps solution (BigPanda, MoogSoft, etc.)
```
Comment (talboren) [inline docs/overview/comparison.mdx:20]: Same as other comment but maybe repeating BigPanda and Moogsoft twice is redundant?
```suggestion
## Keep vs traditional AIOps platforms
```

File: docs/overview/comparison.mdx [hunk 3]
```diff
@@ -4,38 +4,36 @@ title: "Comparison"
...
 
-> Using Grafana Alerting, you create queries and expressions from multiple data sources — no matter where your data is stored — giving you the flexibility to combine your data and alert on your metrics and logs in new and unique ways. You can then create, manage, and take action on your alerts from a single, consolidated view, and improve your team’s ability to identify and resolve issues quickly.
-> [(Grafana docs)](https://grafana.com/docs/grafana/latest/alerting/).
+BigPanda and Moogsoft are our direct competitors. Keep differs because it's open source, which makes integration and piloting easy. Our customer also noticed a significant difference in the alert correlation approach since we at Keep focus less on manual configuration and more on applying state-of-the-art AI models, meaning it's easier for Keep to correlate alerts, which never happened before.
```
Comment (talboren) [inline docs/overview/comparison.mdx:22]: Feels a little bit "testimonial"ish, is it what you meant? tried to apply some changes, but not 100% happy with my changes too 
```suggestion
Keep is an alternative to platforms like BigPanda and Moogsoft. 
We differ with many things but farmost by being open source, which makes integration and piloting easy. 
Customers who tried both traditional platforms and Keep note a significant difference in the alert correlation approach since Keep is a lot less manual and uses state-of-the-art AI models, meaning it's easier for Keep to correlate alerts, which never happened before.
```

File: docs/overview/introduction.mdx [hunk 4]
```diff
@@ -18,8 +18,10 @@ Alerts usually categorized into three different groups:
 Keep helps with every step of the alert lifecycle:
 1. Creation - Keep offers a framework for creating, debugging, and testing alerts through code that scales with your teams.
```
Comment (talboren) [inline docs/overview/introduction.mdx:19]:
```suggestion
```

File: docs/overview/introduction.mdx [hunk 5]
```diff
@@ -18,8 +18,10 @@ Alerts usually categorized into three different groups:
...
 2. Maintenance - Keep integrates with your tools, allowing you to manage all of your alerts within a single interface.
```
Comment (talboren) [inline docs/overview/introduction.mdx:20]: maybe something else instead of maintenance? sounds weird
```suggestion
1. Maintenance - Keep integrates with your tools, allowing you to manage all of your alerts within a single interface.
```

File: docs/overview/introduction.mdx [hunk 6]
```diff
@@ -18,8 +18,10 @@ Alerts usually categorized into three different groups:
...
-3. Noise reduction - By integrating with monitoring tools, Keep can deduplicate and correlate alerts to reduce noise in your organization.
+3. Noise reduction - By integrating with monitoring tools, Keep can deduplicate and correlate alerts to reduce noise in your organization. Rule-based (all distributions), and AI-based (Keep Enterprise only).
```
Comment (talboren) [inline docs/overview/introduction.mdx:21]:
```suggestion
2. Noise reduction - By integrating with monitoring tools, Keep can deduplicate and correlate alerts to reduce noise in your organization. Rule-based (all distributions), and AI-based (Keep Enterprise only).
```

File: docs/overview/introduction.mdx [hunk 7]
```diff
@@ -18,8 +18,10 @@ Alerts usually categorized into three different groups:
...
 4. Automation - [Keep Workflows](/workflows) enable automated alert enrichment and response.
+5. Incident Correlation - Automatically assign alerts to incidents (Keep Enterprise only).
```
Comment (talboren) [inline docs/overview/introduction.mdx:23]: Explain who this differs from #2?

File: docs/overview/introduction.mdx [hunk 8]
```diff
@@ -18,8 +18,10 @@ Alerts usually categorized into three different groups:
...
+6. Summorization - Summary of an incident based on past incidents and the knowledge base (Keep Enterprise only).
```
Comment (talboren) [inline docs/overview/introduction.mdx:24]:
```suggestion
6. Summarization - Summary of an incident based on past incidents and the knowledge base (Keep Enterprise only).
```

Comment (talboren) [reply docs/overview/introduction.mdx:24]: Perhaps add something about triage and root cause analysis?

File: scripts/list_all_integrations.py
Comment (talboren) [inline scripts/list_all_integrations.py:1]: what do we use this specifically for? why not use get_all_providers for instance? :P

**Review by talboren — CHANGES_REQUESTED** [review]
few small comments. this is aweomse 🧡

File: scripts/list_all_integrations.py
Comment (Matvey-Kuk) [reply scripts/list_all_integrations.py:1]: It's for the list on the website: Screenshot 2024-06-17 at 11 09 50

File: docs/overview/introduction.mdx (hunk 7 above)
Comment (Matvey-Kuk) [reply docs/overview/introduction.mdx:23]: Different things people will search for I guess, that's why it worth mentioning them both

File: docs/providers/documentation/slack-provider.mdx
Comment (talboren) [inline docs/providers/documentation/slack-provider.mdx:1]: @Matvey-Kuk can you please revert this change as it violates Slack’s OAuth app policy?

besides that Lgtm!

Comment (Matvey-Kuk) [reply docs/providers/documentation/slack-provider.mdx:1]: Done!

**Review by talboren — APPROVED** [review]
LGTM!
